apm-application-toolkit/apm-toolkit-meter/src/main/java/org/apache/skywalking/apm/toolkit/meter/impl/CounterImpl.java

```diff
@@ -51,30 +48,13 @@ public void increment(double count) {
     /**
      * Get count value
      */
-    public double get() {
+    public double getCount() {
```

Comment:
Why change this name? But such as in `GuageImpl`, still, use `get`? Seems inconsistent to me.

Reply:
Change them to the same function name, using `getCount` to get meter value. I forget to change the `Gauge`, has resolved.